packages/connect-popup/src/view/common.tsx

```diff
@@ -206,3 +166,35 @@ export const postMessageToParent = (message: CoreMessage) => {
         window.postMessage(message, window.location.origin);
     }
 };
+
+let reactRenderIn;
+
+// todo: ConnectUIProps
```

Comment:
is there anything to do with ConnectUIProps?

Reply:
c2f86d72ccffa9bf90b96f92cfd244bff039c878